src/Storefront/Resources/views/storefront/component/line-item/element/image.html.twig

```diff
@@ -1,20 +1,32 @@
 {% block component_line_item_image %}
```

Comment:
I wanted to keep it as close as possible to the image implementation because it's all coming from media. Thats why I've just added the video utility alternatives to sw_thumbnail.